python/sglang/srt/mem_cache/memory_pool_host.py

```diff
@@ -26,9 +26,15 @@
         transfer_kv_per_layer_mla_pf_lf,
         transfer_kv_per_layer_pf_lf,
     )
+if _is_npu:
+    from sgl_kernel_npu.kvcacheio import TransferDirection, transfer_kv_dim_exchange
 
 logger = logging.getLogger(__name__)
 
+SUPPORT_PIN_MEMORY = not _is_npu
+if SUPPORT_PIN_MEMORY:
+    logger.warning("Current platform not support pin_memory")
```

Comment:
1. Do not print annoying messages during the startup
2. The logic is wrong. It acutally supports pin_memory